bundle/manifests/security-profiles-operator-profile_v1_configmap.yaml

```diff
@@ -2,12 +2,9 @@ apiVersion: v1
 data:
   security-profiles-operator.json: |
     {
-      "defaultAction": "SCMP_ACT_ERRNO",
+      "defaultAction": "SCMP_ACT_LOG",
       "architectures": [
-        "SCMP_ARCH_X86_64",
-        "SCMP_ARCH_X86",
-        "SCMP_ARCH_X32",
-        "SCMP_ARCH_AARCH64"
```

Comment:
Combine the prior and this so you have 6 archies listed  (including SCMP_ARCH_PPC64LE
SCMP_ARCH_S390X